crates/iota-rest-kv/src/routes/health.rs

```diff
@@ -0,0 +1,35 @@
+// Copyright (c) 2025 IOTA Stiftung
+// SPDX-License-Identifier: Apache-2.0
+
+use axum::{Json, extract::State, response::IntoResponse};
+use serde::Serialize;
+
+use crate::{kv_store_client::AwsStatus, types::SharedKvStoreClient};
+
+bin_version::bin_version!();
+
+/// Represent a health status response of the REST API server.
+#[derive(Serialize)]
+pub struct HealthResponse {
+    /// Version of the binary.
+    pub version: String,
+    /// The Git hash of the binary.
+    pub git_hash: String,
+    /// The total uptime of the REST API server.
+    pub uptime: String,
+    /// The status of AWS components the REST API rely to properly function.
+    pub aws_status: AwsStatus,
+}
+
+pub async fn health(State(kv_store_client): State<SharedKvStoreClient>) -> impl IntoResponse {
```

Comment:
Maybe just a brief comment to this public function what health it returns.

Reply:
[fixed](https://github.com/iotaledger/iota/pull/5296/commits/bca260fb4f1246d07df63cd2f55b10763744d2c9#diff-c17c6f3893188443a8dea4a29ed01d925b17024f4efc7f2284ffe7d0a2ff180bR24-R27)